.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description of changes:
```

Comment:
I don't like having such a big header level for each section, either in the PR or the issue template. I find it makes the headers overwhelm the actual information. You could keep these at a lower header level, or (my preference) remove the header `##` signs entirely.